packages/completer/src/default/inlinehistoryprovider.ts

```diff
@@ -72,6 +72,11 @@ export class HistoryInlineCompletionProvider
 
     const multiLinePrefix = request.text.slice(0, request.offset);
     const linePrefix = multiLinePrefix.split('\n').slice(-1)[0];
+    const multiLineSuffix = request.text.slice(
+      request.offset,
+      request.text.length
+    );
```

Comment:
I think `indexEnd` can be omitted here:


```suggestion
    const multiLineSuffix = request.text.slice(request.offset);
```